### PR TITLE
pusher: add new pusher debug setting to environment

### PR DESCRIPTION
**Pull request by igorwwwwwwwwwwwwwwwwwwww**
Allows logging pusher events to console as per https://pusher.com/docs/debugging#pusher_logging

### CODE REVIEW

File: app/utils/pusher.js [hunk 1]
```diff
@@ -28,6 +28,15 @@ TravisPusher.prototype.init = function (config, ajaxService) {
   if (config.host) {
     Pusher.host = config.host;
   }
+
+  if (ENV.pusher.debug) {
+    Pusher.log = function (message) {
+      if (window.console && window.console.log) {
```
Comment (lislis) [inline app/utils/pusher.js]: I think the guard is a bit over the top here. We can expect all browsers these days to just work with 
`console.log();`

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline app/utils/pusher.js]: AFAIK `console.log` is sometimes not present when the console is not open? I might be mistaken though. I copy-pated it from the [pusher docs](https://pusher.com/docs/debugging#pusher_logging).

Comment (lislis) [inline app/utils/pusher.js]: Hmm, would be new to me. But if they have it in their docs ¯\_(ツ)_/¯

Comment (igorwwwwwwwwwwwwwwwwwwww) [inline app/utils/pusher.js]: Appears to be the case with some IE versions at least, so probs safer this way: https://stackoverflow.com/a/5473193

**Comment by drogus** [comment]
@igorwwwwwwwwwwwwwwwwwwww do you plan to use it in production? One problem that I see here is that enabling it will enable it for everyone. I'd prefer to add an option to enable it on page load. There's also a way to do it directly from the console, you can run `Pusher.log = (message) => { console.log(message) }` after the page loads.

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
@drogus it's for debugging locally, I found it extremely helpful when debugging things, and not having to paste things into the console on every page load.

If you have a different way of handling code not intended for prod, I'd be happy to change it.

**Comment by drogus** [comment]
@igorwwwwwwwwwwwwwwwwwwww if it's for debugging locally, it's totally fine. I was just worried that it's for production

**Pull request merged**